ccdproc/tests/test_image_collection.py

```diff
@@ -217,6 +217,17 @@ def test_generator_data(self, triage_setup):
         for img in collection.data():
             assert isinstance(img, np.ndarray)
 
+    def test_generator_ccddata(self, triage_setup):
+        from ccdproc import CCDData
+        collection = image_collection.ImageFileCollection(location=triage_setup.test_dir,
+                                             keywords=['imagetyp'])
```

Comment:
the indentation seems wrong. If lines get too long you can also break before the first argument:

```
        collection = image_collection.ImageFileCollection(
            location=triage_setup.test_dir, keywords=['imagetyp'])
```